app/views/candidate_mailer/nudge_unsubmitted_with_incomplete_courses.text.erb

```diff
@@ -18,4 +18,6 @@ Alternatively, call 0800 389 2500 or [chat online](<%= email_link_with_utm_param
 
 Monday to Friday, 8:30am to 5:30pm (except public holidays).
 
+<%= render 'understand_your_professional_strengths' %>
```

Comment:
According to the PR description, should this be `gain_insights_into_life_as_a_teacher`?

Reply:
Checked with rebecca and it should be `gain_insights_into_life_as_a_teacher`, I've changed it now.